website/site/static/admin/config.yml

```diff
@@ -27,6 +27,7 @@ collections: # A list of collections the CMS should be able to edit
           - name: notifications
             label: Notifications
             widget: list
+            disableAdd: false
```

Comment:
We're generally phrasing boolean flags in the positive using snake_case, so rather than `disableAdd: true`, it would be more like `allow_add: false`.